tests/test_integration.py

```diff
@@ -729,6 +729,68 @@ def global_modeling():  ### GLOBAL MODELLING - NO EXOGENOUS VARIABLES
                     fig1 = m.plot(frst)
                     fig2 = m.plot_components(frst)
 
+    def global_modeling_local_normalization():  ### GLOBAL MODELLING - NO EXOGENOUS VARIABLES - LOCAL MODELING
```

Comment:
Could you separate the different global modelling test cases into separate test functions (not nested)?
This would make each one easier to understand.

Reply:
The only problem in this case is to load the peyton dataset over and over. I will separate them anyways. I will also add NROWS to the read_csv call so it makes the testing faster.

Reply:
Sounds good.